.github/actions/next-integration-stat/src/index.ts

```diff
@@ -81,11 +95,16 @@ async function fetchJobLogsFromWorkflow(
       job_id: job.id,
     })
 
+  console.log(
+    'fetchJobLogsFromWorkflow: Trying to get logs from redirect url ',
+    jobLogRedirectResponse.url
+  )
+
   // fetch the actual logs
   const jobLogsResponse = await nodeFetch(jobLogRedirectResponse.url, {
     headers: {
       Accept: 'application/vnd.github.v3+json',
-      Authorization: `token ${token}`,
+      //Authorization: `token ${token}`,
```

Comment:
Is this intentional?

Reply:
Yes, see the comment above

Reply:
https://github.com/vercel/next.js/pull/62665/files#diff-0e0c2f9cb52224d172853cfbed972f9ac25436bf53078850cd7b3a7168fef0e7R49